Client.xcodeproj/project.pbxproj

```diff
@@ -17995,6 +17999,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				CODE_SIGN_ENTITLEMENTS = "";
+				IPHONEOS_DEPLOYMENT_TARGET = 16.4;
```

Comment:
I don't know if this change is necessary. I picked `16.4` instead of the default `16.0` from the build config.